cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -147,6 +147,43 @@ function Dump-DebugInfoToConsole {
   } Catch { }
 }
 
+# Converts the kube-env string in Yaml
+#
+# Returns: a PowerShell Hashtable object containing the key-value pairs from
+#   kube-env.
+function ConvertFrom-Yaml-KubeEnv {
```

Comment:
Did you write this from scratch or is it based on something that already existed?

If the former, I'm a little wary of various corner cases popping up in the future. But I also have no idea about the quality of the powershell-yaml module we're already using.

Reply:
it is based on a code sample about text parsing in powershell. the yaml here is just 'key:value' pairs, we don't really need a full-blown yaml parser, IMO. also the kube env is provided by the control plane, e2e tests should be able to catch the corner cases before they hit production.